public/index.html

```diff
@@ -13,7 +13,14 @@
     <% if (htmlWebpackPlugin.options.NODE_ENV !== 'production' && htmlWebpackPlugin.options.OPTIMIZELY_PROJECT_ID) { %>
       <script src="https://cdn.optimizely.com/js/<%= htmlWebpackPlugin.options.OPTIMIZELY_PROJECT_ID %>.js"></script>
     <% } %>
-
+    
+    <!-- Append the `?module` param to load the module version of `web-vitals` -->
+    <script type="module">
```

Comment:
@long74100 It does not and has since been removed; was leftover from some trial and error to try to get consistent web vital metrics during local development to see if code splitting by route would meaningful impact the web vitals, but I'm not confident in the reported metrics due to slow API response times (e.g., ecommerce).